tools/create-translation-issues.sh

```diff
@@ -94,7 +96,7 @@ We are getting ready for the ${RELEASE} release and have created the [${TRANSLAT
 
 We would be happy if you updated the translations for **${LANG}** by **${DEADLINE}**.
 
-Please
+Please either [update the translations on Weblate](https://hosted.weblate.org/projects/jamulus/) or use git:
```

Comment:
```suggestion
Please either [update the translations on Weblate](https://hosted.weblate.org/projects/jamulus/) or use Git:
```

Reply:
The Git Manpages resource begs to differ
https://hosted.weblate.org/zen/git-manpages/translations/en/?offset=1&q=%22+Git+%22&sort_by=-priority%2Cposition&checksum=

Reply:
Fair point.  I guess we'd rather Jamulus were "Jamulus" everywhere when referred to by name.

Reply:
Done.